experimental/codepointtrie/src/codepointtrie.rs

```diff
@@ -0,0 +1,906 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::error::{Error, FromDeserializedError};
+use crate::impl_const::*;
+use std::marker::PhantomData;
+use zerovec::ZeroVec;
+
+// Enums
+
+/// The width of the elements in the data array of a CodePointTrie.
```

Comment:
Nit: Here and elsewhere: Use intra-doc links.  You can preview how they look with `cargo doc --open`.
```suggestion
/// The width of the elements in the data array of a [`CodePointTrie`].
```